api/src/main/java/org/eclipse/microprofile/config/Config.java

```diff
@@ -124,7 +134,60 @@
     <T> Optional<T> getOptionalValue(String propertyName, Class<T> propertyType);
 
     /**
-     * Return a collection of property names.
+     * Create a {@link ConfigAccessor} to access the underlying configuration.
+     *
+     * @param propertyName the property key
+     * @return a {@code ConfigAccessor} to access the given propertyName
+     */
+    ConfigAccessor<String> access(String propertyName);
```

Comment:
The String should be T

Reply:
gnah, resolving ditches the commtent :(

So here again: My version is correct. Default is `ConfigAccessor<String>`
If one wants another type then use `as(SomeType.class)` which will then return a `ConfigAccessor<SomeType>`